src/auth/oauth2client.ts

```diff
@@ -475,6 +481,11 @@ export interface OAuth2ClientOptions extends AuthClientOptions {
    * The allowed OAuth2 token issuers.
    */
   issuers?: string[];
+  /**
+   * The client_authentication choice from baisc/post/none
+   * https://docs.authlib.org/en/latest/client/oauth2.html#client-authentication
```

Comment:
Why is this linking to a specific python implementation? I think we should link to the rfc directly instead ([RFC7591](https://datatracker.ietf.org/doc/html/rfc7591#section-2))

Reply:
Yeah, let me change into https://datatracker.ietf.org/doc/html/rfc7591#section-2